lib/rubocop/cop/style/mixin_usage.rb

```diff
@@ -48,7 +48,7 @@ class MixinUsage < Cop
 
         def_node_matcher :include_statement, <<-PATTERN
           (send nil? ${:include :extend :prepend}
-            (const nil? _))
+            (const _ _))
```

Comment:
We can replace `(const _ _)` with `const`

Reply:
I fixed it. Thanks!